dataprofiler/tests/data_readers/test_csv_data.py

```diff
@@ -577,36 +541,11 @@ def test_allowed_data_formats(self):
                 self.assertEqual(input_data_obj.data_format, data_format)
                 data = input_data_obj.data
                 if data_format == "dataframe":
-                    import pandas as pd
-
                     self.assertIsInstance(data, pd.DataFrame)
                 elif data_format in ["records", "json"]:
                     self.assertIsInstance(data, list)
                     self.assertIsInstance(data[0], str)
 
-    def test_allowed_data_formats_when_sampled(self):
```

Comment:
I don't think the functionality covered in this test is affected by reservoir sampling.

Reply:
`input_data_obj = Data(input_file["path"], options={"sample_nrows": 100})`